native/src/components/OpeningHours.tsx

```diff
@@ -56,8 +56,7 @@ const OpeningHours = ({
   const { t } = useTranslation('pois')
 
   // The opening hours loaded from the cms are ordered according to the german weekday order
-  // @ts-expect-error Interface has wrong type, supplying true is necessary and works
-  const weekdays = moment.localeData('de').weekdays(true)
+  const weekdays = Info.weekdays('long', { locale: 'de' })
```

Comment:
Somehow for me only `weekdaysFormat` works with the german locale. 
Does this work for you? Could be because of nodes intl support.

Reply:
@sarahsporck yes, it is working